README.md

```diff
@@ -1 +1,48 @@
-# Portfolio
\ No newline at end of file
+
+# Microverse Portfolio Project
+> Portfolio project created using Figma design.
+### Mobile View
+![screenshot](mobile.png)
+### Desktop View
+![screenshot](desktop.png)
+
+
+## Built With
+- Html
+- CSS
+- GITHub
+
+## Getting Started
+ To get a local copy up and running follow these simple example steps.
+- Clone this repository or download the Zip folder:
+git clone https://github.com/mr-anuj/Portfolio.git
+- Navigate to the location of the folder in your machine
+
+### Prerequisites
+- Web browser (like Google Chrome, Opera...).
+- Code editor (like Atom, VScode...).
+
+## Setup
+To setup the project locally: click on "Code" (dropdown list) then "Download ZIP".
+
+## Deployment
+To run the project open the downloaded folder (after extraction) then run the "index.html" file in your browser.
+
+## Authors
+
+👤 **Anuj Chettri**
+
+- GitHub: [@githubhandle](https://github.com/mr-anuj)
+
+## Contributing
+Contributions, issues, and feature requests are welcome!
+
+## Show your support
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+- Hat tip to Microverse's code reviewers and to everyone who reviewed my project and made suggestions.
+- The website design is from [@Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)
+## License
+This project is [MIT]() licensed.
+
```

Comment:
- [ ] Please update your Readme, it will be nice to add and update this section(`License`) with the type of License and a link to the license file, as this will make your readme more informative and professional. Check out this [template](https://github.com/microverseinc/readme-template) for more clarifications
-  Please, consider fixing the broken link to the MIT.md file 🥇